app/src/main/kotlin/at/bitfire/davdroid/ui/account/GetServiceCollectionPagerUseCase.kt

```diff
@@ -72,6 +71,6 @@ class GetServiceCollectionPagerUseCase @Inject constructor(
                 else
                     dataFlow
             }
-        }.flattenConcat()
+        }.flatMapLatest { it }
```

Comment:
Just a small comment: I think we can combine the two `if`s into one, then the flow is not mapped twice for nothing in most cases (and we can omit one `else`).

Should work, shouldn't it?

Reply:
I have lifted out the if, so that in most cases (`forceReadOnlyAddressBooks == false`) the whole double-map thing isn't even called.